src/HLAdapter.hx

```diff
@@ -97,7 +98,9 @@ class HLAdapter extends DebugSession {
 	}
 
 	override function setExceptionBreakPointsRequest(response:SetExceptionBreakpointsResponse, args:SetExceptionBreakpointsArguments) {
-		dbg.breakOnThrow = args.filters.indexOf("all") >= 0;
+		if( Sys.systemName() != "Mac" ) { // TODO: Fix issue with corrupted memory on Mac
```

Comment:
What is this TODO about ?

Reply:
Currently this means that "Stop on all exceptions" breakpoints aren't working yet. There is some issue in the underlying C implementation of the debugger that still need to be fixed  in the future (see https://github.com/HaxeFoundation/hashlink/pull/367).

Can you please report any issues or let me know if everything is working as you would expect after using it for a while, so that there is some proof of readiness before a potential merge of the PR.

Thanks!